src/App/Program.fs

```diff
@@ -459,6 +462,21 @@ type MissionOptions
              Required = false)>]
     member self.NumRuns = numRuns
 
+    [<Option("num-pregenerated-txs",
+             HelpText = "Number of transactions to pregenerate for max TPS tests",
+             Required = false)>]
+    member self.NumPregeneratedTxs = numPregeneratedTxs
+
+    [<Option("enable-parallel-apply",
```

Comment:
I think this flag is currently broken. When set, I think we still use SQLite DBs, so core workers will just throw on startup

Reply:
ehhh yea, this belong to a different PR (I have another branch that's using Postgres). I'll just add the option piping there